mobile-app/app/api/transaction/dfi_converter.ts

```diff
@@ -38,21 +38,16 @@ export async function dfiConversionSigner (account: WhaleWalletAccount, amount:
 }
 
 export function dfiConversionCrafter (amount: BigNumber, mode: ConversionMode, onBroadcast: () => any, submitButtonLabel?: string): DfTxSigner {
+  const [symbolA, symbolB] = mode === 'utxosToAccount' ? ['UTXO', 'DFI'] : ['DFI', 'UTXO']
   return {
     sign: async (account: WhaleWalletAccount) => await dfiConversionSigner(account, amount, mode),
     title: translate('screens/ConvertConfirmScreen', 'Converting DFI'),
-    description: translate('screens/ConvertConfirmScreen', 'Converting {{amount}} {{symbolA}} to {{symbolB}}', {
-      amount: amount.toFixed(8),
-      ...(mode === 'utxosToAccount'
-        ? {
-          symbolA: 'UTXO',
-          symbolB: 'Token'
-        }
-        : {
-          symbolA: 'Token',
-          symbolB: 'UTXO'
-        })
-    }),
+    drawerMessages: {
+      preparing: translate('screens/OceanInterface', 'Preparing to convert…'),
+      waiting: translate('screens/OceanInterface', 'Converting {{symbolA}} to {{symbolB}}', { symbolA, symbolB }),
```

Comment:
This is the text for converting
![image](https://user-images.githubusercontent.com/13292662/183341998-baa082a8-a66d-4686-a361-e0b2cb9449cd.png)
![image](https://user-images.githubusercontent.com/13292662/183342008-8cf23de6-ff41-4c54-a050-a35483163fc8.png)